Library/Formula/pgroonga.rb

```diff
@@ -0,0 +1,16 @@
+class Pgroonga < Formula
+  desc "An extension for PostgreSQL to use Groonga as an index."
+  homepage "https://pgroonga.github.io/"
+  url "http://packages.groonga.org/source/pgroonga/pgroonga-0.6.0.tar.gz"
+  sha256 "29a631f68013bba19321beac0d7b933eafce60aec1597aa8fff0b0898ff2a73d"
+
+  depends_on "cmake" => :build
+  depends_on "pkgconfig" => :build
+  depends_on "postgresql"
+  depends_on "groonga"
+
+  def install
+    system "make"
+    system "make", "install"
+  end
+end
```

Comment:
Hmm..., it's a bit difficult to test.
`PGroonga` is shared object, not executables. It does not have entry point.
And now, PGroonga does not have executable example, though.